Automattic/wp-calypso
# Stats: Add banner for Parse.ly

#### Changes proposed in this Pull Request

* Replace Cloudflare and money upsells with Parse.ly banner

**Before**

![wordpress com_stats_day_sixhours wpcomstaging com](https://user-images.githubusercontent.com/2124984/132048741-b1e4abf1-a4b5-481b-ac04-2b487218c3ef.png)

**After**

![calypso localhost_3000_stats_day_sixhours wpcomstaging com](https://user-images.githubusercontent.com/2124984/132383878-6d8bcdc7-a600-43d3-9a36-aae1e2d97ff3.png)

#### Testing instructions

* Switch to this PR
* Navigate to `/stats/[site]`
* Scroll down - there should be a Parse.ly banner at the bottom of the screen
* Make sure "Learn more" links to `http://parse.ly`
* Check to make sure Tracks events fire when you view and click on the banner - all prefixed with `calypso_stats_parsely_banner_*`
